src/main/java/org/fcrepo/auth/webac/URIConstants.java

```diff
@@ -26,7 +26,10 @@
  *      http://www.w3.org/wiki/WebAccessControl/Vocabulary</a>
  * @see <a href="http://www.w3.org/ns/auth/acl">http://www.w3.org/ns/auth/acl</a>
  */
-public abstract class URIConstants {
+public class URIConstants {
+
+    private URIConstants() {
+    }
```

Comment:
sorry to be picky here -- I believe the convention is to declare constructors *after* the instance variables. Can you move this to the end of the class decl?